issue_template.md

```diff
@@ -0,0 +1,31 @@
+## Description
+
+Briefly describe the issue
+
+## Chef Version
+
+Tell us which version of chef-client you are using (This repo is only for chef-client bugs).
+
+## Platform Version
+
+Tell us which Operating System distribution and version chef-client is running on.
+
+## Replication Case
+
+Tell us what steps to take to replicate your problem.  See [How to create a Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve)
+for information on how to create a good replication case.
+
+## Error Output
+
+The complete output of the chef-client run or a link to a gist of it, if there is one.
+
+The debug output (chef-client -l debug) may be useful, but please link to a gist, or truncate it.
+
+## Stacktrace
+
+Please include the stacktrace.out output or link to a gist of it, if there is one.
+
+### NOTE: CHEF CLIENT BUGS ONLY
```

Comment:
I think we can fix that the first time users ask 'where do kinife bugs go?'  There's also chef-apply and chef-zero should go in the chef-zero repo, while local-mode should go here.

Its also particularly hard because knife plugins need to go to the plugins and knife bugs need to go here, so wording would need to get that right, too.

But really 90+% of the misfiled bugs are server bugs winding up in this repo.  I'd prefer to keep it simple until someone posts that they're confused.